services/user_service.py

```
@@ -0,0 +1,67 @@
+
+            image_url = f"/{file_path}"
+
+        user_id = self.user_repo.get_next_id()
+
+        new_user = {
+            "id": user_id,
+            "email": signup_data.email,
+            "nickname": signup_data.nickname,
+            "password": hash_password(signup_data.password),
+            "profile_image": image_url,
+            "created_at": datetime.now().isoformat()
```

Ruff (0.14.11)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)